mods/stairs/init.lua

```diff
@@ -50,20 +50,25 @@ end
 
 function stairs.register_stair(subname, recipeitem, groups, images, description, sounds)
 	-- Set backface culling and world-aligned textures
+	local is_facedir = minetest.registered_nodes[recipeitem].paramtype2 ==
```

Comment:
This might actually break some mods which register stairs before the regular node. The function here did not require any order.
Needs testing with many (exotic) mods (dreambuilder, mineclone) to ensure it does not affect too many mods.

Reply:
Maybe then make this an additional field in stair registering, for more flexibility?